codalab_service.py

```diff
@@ -716,6 +716,12 @@ def build_image(self, image):
         else:
             build_args = ''
 
+        if image == 'frontend' and not self.args.dev:
+            CODALAB_RECAPTCHA_SITE_KEY = os.environ['CODALAB_RECAPTCHA_SITE_KEY']
```

Comment:
Add a comment mentioning that we need to do this to pass along the frontend variables during build time.

Reply:
Comment added.